LondonBritishLibrary/orient/BLorient784.xml

```diff
@@ -0,0 +1,633 @@
+                          <msItem xml:id="ms_i1.70">
+                            <locus from="208r"></locus>
+                            <title type="complete" ref="LIT1586Hayman#Contents"></title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <colophon xml:id="coloph1" xml:lang="gez"><locus target="#209r"></locus>
+                          በሥምረተ፡ እግዚአብሔር፡ ተጽሕፈ፡ ዝንቱ፡ ሃይማኖተ፡ አበው፡ ቅዱሳን፡ በመዋዕለ፡ ንጉሥነ፡ መፍቀሬ፡
+                          እግዚአብሔር፡ <persName ref="PRS5616IyasuI">አድያም፡ ሰገድ።</persName>
+                          ወበጸጋ፡ ጥምቀት፡ ዘተጸውዓ፡ በስመ፡ መሲሕ፡ ኢያሱ።
+                          በአስተሐምሞቱ፡ ወበጻሕቄ፡ ለ<del rend="erasure"></del> ወተፈጸመ፡ እምአመ፡ ነግሡ፡ በ፲ወ፫፡ ዓመት፡ በዘመነ፡ ሉቃስ፡
+                          ወንጌላዊ። ወ<date calendar="world" when-custom="7187" when="1695">በዓመተ፡ ዓለም፡ ፸፻ወ፩፻፹ወ፯።</date>
+                          ባሕቱ፡ ፈጣሬ፡ ዓለማት፡ ያኑኅ፡ መዋዕሊሆሙ፡ ለንጉሥነ፡
+                          ዳግማይ፡ ቈስጠንጢኖስ፡ ዘሐደስዋ፡ ለብሊት፡ ዓለም፡ በርትዕት፡ ሃይማኖት። ወይዕቀቦሙ፡ ከመ፡
+                          ብንተ፡ ዓይን፡ ወያግርር፡ አጽራሪሆሙ፡ ታሕተ፡ እግረ፡ ሥልጣኖሙ፡ መሲሓዊት፡ እስከ፡ እስትንፋስ፡
+                          ደኃሪት። አሜን።
+                          </colophon>
+                        </msItem>
+
+
+
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">209</measure>
+                                    <measure unit="leaf" type="blank">7</measure>
+
+                                    <dimensions type="outer" unit="in">
+                                        <height>14.5</height>
+                                        <width>12.875</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="27">
+                                </layout>
+                            </layoutDesc>
+
+                        </objectDesc>
+
+                        <handDesc>
+
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Written in fine hand.</desc>
+                                <date when="1695">Dated in the colophon <ref target="#coloph1"></ref>. </date>
+
+                            </handNote>
+
+                        </handDesc>
+
+                        <additions>
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#12 #209"></locus>
+                              <desc>Amharic deeds of the reign of <persName ref="PRS2377Bakaffa"></persName>,
```

Comment:
<desc type="LandGrant"?

Reply:
Hmm, I am not sure, there is no specification. I have therefore added type=Record